nodes/clusters/nodes-cluster-enabling-features.adoc

```diff
@@ -13,12 +13,11 @@ include::modules/nodes-cluster-enabling-features-about.adoc[leveloffset=+1]
 .Additional resources
 
 * For more information on the features activated by the `TechPreviewNoUpdate` feature gate, see the following topics:
-
-** xref:../../storage/container_storage_interface/persistent-storage-csi-azure.adoc#persistent-storage-csi-azure-disk[Azure Disk CSI Driver Operator]
-** xref:../../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-vsphere[VMware vSphere CSI Driver Operator]
+** xref:../../storage/container_storage_interface/persistent-storage-csi-azure-file.adoc#persistent-storage-csi-azure-file[Azure File CSI Driver Operator]
 ** xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration]
 ** xref:../../operators/operator-reference.html#cluster-cloud-controller-manager-operator_platform-operators-ref[Cluster Cloud Controller Manager Operator]
-** xref:../../support/remote_health_monitoring/insights-operator-simple-access.adoc#insights-operator-simple-access[Importing simple content access certificates with Insights Operator ]
+** xref:../../cicd/builds/build-strategies.html#builds-using-build-volumes_build-strategies-s2i[Source-to-image (S2I) build volumes] and xref:../../cicd/builds/build-strategies.html#builds-using-build-volumes_build-strategies-docker[Docker build volumes].
```

Comment:
I dont think you need the period here. It is a longer list item but still not a complete sentence